packages/components/app/styles/components/copy/snippet.scss

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// COPY > SNIPPET
+//
+@use "../../mixins/focus-ring" as *;
+
+.hds-copy-snippet {
+  @include hds-focus-ring-with-pseudo-element();
+
+  display: flex;
+  gap: 8px;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  max-width: 100%;
```

Comment:
Since you define the width as 100% already, you shouldn't need to add a max-width also.